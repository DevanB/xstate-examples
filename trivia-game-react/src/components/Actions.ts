import tw from 'tailwind.macro'

export const Actions = tw.div`flex flex-col md:flex-row w-full md:w-1/2 justify-around`
