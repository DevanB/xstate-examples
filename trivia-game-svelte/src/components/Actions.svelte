<div class="actions">
  <slot></slot>
</div>

<style>
.actions {
  @apply flex flex-col w-full justify-around;
}

@screen md {
  .actions {
    @apply flex-row w-1/2
  }
}
</style>
