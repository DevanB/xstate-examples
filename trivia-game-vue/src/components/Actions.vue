<template>
  <div class="actions">
    <slot></slot>
  </div>
</template>

<script>
export default {
  name: "Actions"
}
</script>

<style>
.actions {
  @apply flex flex-col w-full justify-around;
}

@screen md {
  .actions {
    @apply flex-row w-1/2
  }
}
</style>
