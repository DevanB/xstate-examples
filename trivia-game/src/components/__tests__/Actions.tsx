import {render} from '@testing-library/react'
import React from 'react'
import {Actions} from '../Actions'

describe('<Actions/>', () => {
  it('should match snapshot', () => {
    const {container} = render(<Actions />)
    expect(container).toMatchSnapshot()
  })
})
