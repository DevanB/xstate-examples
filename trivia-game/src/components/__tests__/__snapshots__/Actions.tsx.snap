// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Actions/> should match snapshot 1`] = `
<div>
  <div
    class="Actions-sc-1u0w5ed-0 cZbJYt"
  />
</div>
`;
